csprng-real.php

```diff
@@ -1,5 +1,5 @@
 <?php
-
+require "csprng.php";
```

Comment:
I don't want to include this since `csprng-real.php` is the file I run after compiling the [PHP-src patch](https://github.com/php/php-src/pull/1119) to test it. :)